dts/bindings/pwm/telink,b91-pwm.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2021, Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+
+description: Telink B91 PWM
+
+include: [pwm-controller.yaml, base.yaml]
+
+compatible: "telink,b91-pwm"
+
+properties:
+
+    pinctrl-0:
+      type: phandles
+      required: true
+
+    clock-frequency:
+      type: int
+      required: true
+      description: Default PWM Peripheral Clock frequency in Hz (is used if 32K Clock is disabled)
+
+    clk32k-ch0-enable:
```

Comment:
It seems like it might be more ergonomic to make these optional booleans. Did you consider that? If so, why did you choose a required int?

Reply:
Done.
Reworked to use booleans.